.devops/azure-templates/maven-github-release.yml

```diff
@@ -0,0 +1,157 @@
+# Node Github Relase steps
```

Comment:
To delete. Pipeline should use `/.devops/azure-templates/gradle-github-release.yml`